$NetBSD: patch-ae,v 1.6 2007/01/19 08:02:33 rillig Exp $

Tags: from-upstream

Correctly set the terminal size on FreeBSD and NetBSD.

--- src/command.c.orig	2005-06-21 22:08:16.000000000 +0200
+++ src/command.c	2007-01-19 08:56:52.000000000 +0100
@@ -1085,6 +1085,8 @@ run_command(char *argv[])
 #endif				/* SRIOCSREDIR */
 	}
 
+	tt_winsize(0);		/* set window size */
+
     /* reset signals and spin off the command interpreter */
 	signal(SIGINT, SIG_DFL);
 	signal(SIGQUIT, SIG_DFL);
